core/main/src/firebolt/firebolt_gateway.rs

```diff
@@ -291,11 +291,6 @@ impl FireboltGateway {
             .metrics
             .add_api_stats(&request_c.ctx.request_id, &request_c.method);
 
-        let metrics_timer = TelemetryBuilder::start_firebolt_metrics_timer(
```

Comment:
I'm trying to understand the implications of re/moving the metrics timer logic. Doesn't this mean we're no longer accounting for all of the time spent processing requests, e.g. routing, permission checking, etc.?

Reply:
Observability was already removed which uses Timer, Counter this code was just setting these values